src/cache.js

```diff
@@ -0,0 +1,23 @@
+const cache = require("@actions/cache");
+const github = require("@actions/github");
+const os = require("os");
+
+const CACHE_PATHS = ["~/.foundry/cache/rpc"];
```

Comment:
Does this also cache any etherscan queries? Should we add support for that?

Reply:
Good idea, I didn't think about Etherscan. Opened an issue for tracking:

https://github.com/foundry-rs/foundry-toolchain/issues/20